src/site/content/en/blog/multi-screen-window-placement/index.md

```diff
@@ -338,9 +394,9 @@ You can play with the [demo][demo] embedded below, or see its [source code][demo
 
 <!-- Copy and Paste Me -->
 <div class="glitch-embed-wrap" style="height: 800px; width: 100%;">
-  <iframe src="https://glitch.com/embed/#!/embed/window-placement?path=iframe.html&previewSize=100"
+  <iframe src="https://window-placement.glitch.me/"
```

Comment:
The feature now has permission policy support, so if an iframe will actually attempt to use the API, it will the iframe will need an "allow='window-placement'" attribute.

Reply:
Hmm, clicking elmer just opens the demo in a separate tab, so perhaps we don't need to support the iframe itself using getScreens(), etc.

Reply:
I have (had) logic in the demo that would recognize when it was iframed and open in a new tab. With the new permission policy integration we can get rid of this.